docs/src/prototypes/alerts/AnimatedBannerAlert.tsx

```diff
@@ -20,17 +20,17 @@ const BannerAlert: React.FunctionComponent<BannerAlertProps> = props => {
 
         if (oof) {
           return {
-            color: siteVars.orchid,
-            backgroundColor: '#F9F5F8',
-            borderColor: '#DCADC7',
+            color: siteVars.colors.pink[600],
+            backgroundColor: '#F9F5F8', // where is this color coming from?
```

Comment:
@mnajdova these colors are not yet introduced by design team, but they are correct, you can sync with them maybe they have names for them already

Reply:
I wouldn't add them currently as those are used in the prototypes, but once we have them in the variables, we may ask for updates. @codepretty do you have some input on this one?